symbology/picture-marker-symbol/src/main/java/com/esri/samples/picture_marker_symbol/PictureMarkerSymbolSample.java

```diff
@@ -46,6 +45,12 @@
   private File orangeSymbolPath;
   private GraphicsOverlay graphicsOverlay;
 
+  // keeps loadables in scope to avoid garbage collection
+  private ArcGISMap map;
+  private PictureMarkerSymbol campsiteSymbol;
```

Comment:
I've just learnt that PictureMarkerSymbol are Loadable! Every day is a school day!
Are they affected by this problem as well, even though we don't add a `DoneLoadingListener`?

Reply:
yes, an `addDoneLoadingListener` is used later in the method `placePictureMarkerSymbol` where a picture marker symbol is a parameter :)

Reply:
Woops, I obviously completely missed that! Trying to do too many things at once, and failing a bit.